docs/source/en/using-diffusers/diffedit.md

```diff
@@ -92,7 +96,7 @@ image = pipeline(
     image_latents=inv_latents,
     negative_prompt=source_prompt,
 ).images[0]
-image.save("edited_image.png")
+make_image_grid([raw_image.resize((512, 512)), Image.fromarray((mask_image.squeeze()*255).astype("uint8"), "L").resize((512, 512)), image.resize((512, 512))], rows=1, cols=3)
```

Comment:
This looks a bit messy and harder to read, can we simplify it to something like: `make_image_grid(raw_image, mask_image, output_image)`?

Reply:
Done.